conf/parametersSchema.neon

```diff
@@ -143,6 +143,25 @@ parametersSchema:
 				?identifiers: listOf(string())
 				?reportUnmatched: bool()
 			])
+			structure([
+				rawMessage: string()
+				?identifier: string()
+				?path: string()
+				?reportUnmatched: bool()
+			]),
+			structure([
+				rawMessage: string()
+				count: int()
+				path: string()
+				?identifier: string()
+				?reportUnmatched: bool()
+			]),
+			structure([
+				rawMessage: string()
+				paths: listOf(string())
+				?identifier: string()
+				?reportUnmatched: bool()
+			]),
```

Comment:
This is quickly getting out of hand. You're missing some combinations, like with `identifiers` key.

Maybe we could validate these entries manually in https://github.com/phpstan/phpstan-src/blob/74ac94c93e580bdc5b39b6c8a704508fab927251/src/DependencyInjection/ContainerFactory.php#L303 instead of trying to have a huge declarative schema describing all combinations.

Reply:
> This is quickly getting out of hand. You're missing some combinations, like with identifiers key.

`identifiers` didn't exist when I created the pull request. Added them now.

> Maybe we could validate these entries manually in [...]

Should this be done here in the pr? Also reducing the existing  combinations?

Reply:
You can try sending a separate PR that would move the validation. And once it's merged, rebase this PR and adjust the new logic 😊 Thank you.

Reply:
I'd rather them to be validated in ContainerFactory.